server/routes/aws/cloudwatch_tasks.py

```
@@ -0,0 +1,545 @@
+        previous_state: str, reason: str, account_id: str, region: str,
+    ):
+        self.alarm_name = alarm_name
+        self.alarm_arn = alarm_arn
+        self.state_value = state_value
+        self.previous_state = previous_state
+        self.reason = reason
+        self.account_id = account_id
+        self.region = region
+
+
+def _persist_alarm(
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_persist_alarm`

(ANN202)